perf/benchmark.js

```diff
@@ -22,8 +23,24 @@ const parseTemplate = require('../lib/parse-template')(['fragment'], []);
 const requests = new WeakMap();
 const fragments = new WeakMap();
 const Tailor = require('../');
+const buildBlock = require('../lib/block');
+
+const handleTag = (request, context, tag) => {
+    if (tag && tag.name === 'switcher') {
+        const stream = buildBlock(request, context);
+        process.nextTick(() => {
```

Comment:
is it required? `buildBlock` a sync function

Reply:
I was thinking about kinda real-time example when data is being pushed into this "block" sometime later on (but without big impact on performance).

Reply:
Hmmm true, then lets introduce artificial delay and check if back pressure is handled properly.. `setTimeout` would suffice.